.github/workflows/main.yml

```diff
@@ -7,6 +7,122 @@ on:
     branches: master
 
 jobs:
+  Tests_pandas:
+    name: Tests pandas
+    runs-on: ubuntu-latest
+    env:
+      BACKENDS: "pandas csv parquet hdf5"
+    strategy:
+      matrix:
+        python_version: ["3.7", "3.8"]
+    steps:
+    - name: checkout
+      uses: actions/checkout@v1
+
+    - name: set up environment
+      run: ./ci/setup_env.sh "${{ matrix.pytest_version }}" "$BACKENDS"
+      shell: bash
+
+    - name: run tests
+      run: ./ci/run_tests.sh "not udf and not mysql and not clickhouse and not impala and not kudu and not hdfs and not omniscidb and not spark and not pyspark"
+      shell: bash
+
+  Tests_sql:
+    name: Tests SQL
+    runs-on: ubuntu-latest
+    env:
+      BACKENDS: "postgres sqlite" # TODO add mysql
+    strategy:
+      matrix:
+        python_version: ["3.7", "3.8"]
+    services:
+      postgres:
+        image: shajekpivotal/ibis-docker-postgres-9.5
+        ports:
+          - 5432:5432
+        env:
+          POSTGRES_PASSWORD: ''
+    steps:
+    - name: checkout
+      uses: actions/checkout@v1
+
+    - name: set up environment
+      run: ./ci/setup_env.sh "${{ matrix.pytest_version }}" "$BACKENDS"
+      shell: bash
+
+    - name: run tests
+      run: ./ci/run_tests.sh "not udf and not mysql and not clickhouse and not impala and not kudu and not hdfs and not omniscidb and not spark and not pyspark and not parquet" # TODO remove mysql
```

Comment:
would be even cooler to pass in the backends and have that work but cool as is

Reply:
I thought exactly the same. I had a look and I think it's actually quite simple to implement, and this will be very clear.